packages/features/bookings/Booker/Booker.tsx

```diff
@@ -79,24 +83,47 @@ const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: Book
   }, [isMobile, setLayout, layout]);
 
   useEffect(() => {
-    if (event.isLoading) return setBookerState("loading");
+    if (isLoading) return setBookerState("loading");
     if (!selectedDate) return setBookerState("selecting_date");
     if (!selectedTimeslot) return setBookerState("selecting_time");
     return setBookerState("booking");
-  }, [event, selectedDate, selectedTimeslot, setBookerState]);
+  }, [selectedDate, selectedTimeslot, setBookerState, isLoading]);
 
   useEffect(() => {
     if (layout === "mobile") {
       timeslotsRef.current?.scrollIntoView({ behavior: "smooth" });
     }
   }, [layout]);
 
+  const profileName = event?.profile?.name ?? "";
+  const profileImage = event?.profile?.image;
+  const title = event?.title ?? "";
   return (
     <>
       {/*
         If we would render this on mobile, it would unset the mobile variant,
         since that's not a valid option, so it would set the layout to null.
       */}
+
+      <HeadSeo
```

Comment:
Perhaps it makes most sense to do a separate query on the server (that only queries the SEO data?) and isn't reused on the client. That way we don't have a flash, but are also able to render the SEO data without querying unnecessary data on the server that then isn't used.